tests/testUtils.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const {
+  inputMeta,
+  inputValidation,
+  inputMapping,
+  inputConstants
+} = require('../src/constants')
+
+const pathToDirectory = path.resolve('endpoints', 'integrationTest')
+const metaFilePath = path.resolve(pathToDirectory, inputMeta)
+const inputValidationFilePath = path.resolve(pathToDirectory, inputValidation)
+const inputMappingFilePath = path.resolve(pathToDirectory, inputMapping)
+const inputConstantsFilePath = path.resolve(pathToDirectory, inputConstants)
+
+const metaFileContent = {
+  name: 'IntegrationTest',
+  endpoint: {
+    pattern: '/integrationTest'
+  },
+  transformation: {
+    input: 'XML',
+    output: 'JSON'
+  }
+}
+
+const inputValidationFileContent = {
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string'
+    },
+    surname: {
+      type: 'string'
+    },
+    attributes: {
+      type: 'array',
+      items: {
+        type: 'string'
+      }
+    }
+  },
+  required: ['name']
+}
+
+const inputMappingFileContent = {
+  input: {
+    name: 'firstName',
+    surname: 'lastName',
+    'attributes': 'character'
```

Comment:
We can remove the string declaration around this property key

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/27/commits/134fb5baed0496745fe850b572fd58e4eec091b1